inst/tests/tests.Rraw

```diff
@@ -3667,6 +3667,27 @@ test(1100, dt1[dt2,roll=-Inf,rollends=c(FALSE,TRUE)]$ind, INT(NA,NA,1,2,2,2,2,2,
   DT = data.table(a=sample(10), b=2013:2014, variable=rep(c("c", "d"), each=10), value=runif(20))
   test(1102.18, names(dcast(DT, a ~ ... + b, value.var="value")), c("a", "c_2013", "c_2014", "d_2013", "d_2014"))
 
+  # issue 5824 - tests for "value.var.in.dots", "value.var.in.LHSdots", "value.var.in.RHSdots" arguments of dcast
+  DT = data.table(index = c("a","b"), x = 1:2)
+  set.seed(1)
+  DT2 = data.table(index = c("a","b"), x = 1:2, y = rnorm(2))
+  DT3 = data.table(year = c(rep(1986,4), rep(1987,3), rep(1988,2), 1989:1991), continent = rep(c("Europe","Asia"), each = 6), country = rep(c("Sweden","Germany","France","India","Japan","China"), each = 2))
```

Comment:
I would recommend avoiding variable names with numbers in them like `DT3` and `DT3_dcasted2` -- can you please fix by changing numbers to informative names? (something that communicates expectation / intent would be appreciated)
also you may consider changing DT2 to DT, but moving the definition down below to where it is actually used (and after the tests involving the first DT)

Reply:
Thanks! I follow both of your suggestions. On one hand, reorganizing the code to have the definitions only when they will be used, and on the other hand with some names a bit more informative. I will push in a few seconds.